src/lib/mina_base/test/zkapp_command/call_forest.ml

```diff
@@ -0,0 +1,320 @@
+open Core_kernel
+open Signature_lib
+open Mina_base
+open Mina_base.Zkapp_command.Call_forest
+
+(* TODO: move generators to the library once the code isn't actively being
+     worked on *)
+module Tree = struct
+  include Tree
+
+  module Stable = struct
+    include Stable
+
+    module V1 = struct
+      include V1
+
+      let quickcheck_generator account_update_gen account_update_digest_gen
```

Comment:
In the rest of the code, the name for Quickcheck generators is `gen`, so I'd suggest using that name

Reply:
Should we leave `quickcheck_generator` as an alias for `gen` (as in `account_update.ml`) to allow the usage of `ppx_quickcheck`?

Reply:
Sure.